### DISCUSSION TITLE
read keypress every tot ms

### DISCUSSION THREAD

**Discussion by emanuelmanea** [rxhanson/Rectangle#443] [Ideas]
Hello,

Maybe the feature is there but I couldn't find how to activate it.
I use "Make Smaller/Lager" a lot and I'd find it usefull if the size kept on increasing without lifting the fingers from the keycombo, until the keycombo is no longer pressed.

Let me know if this is possible and how to achieve it.
If possible it makes sense to allow customizing the frequency used to read the keycombo.

Thanks and regards,
Emanuel

**Comment by rxhanson** [comment]
Right now this is a limitation with the shortcut recorder used in Rectangle (MASShortcut) because it does not have functionality for listening to keyboard shortcut down and up. The options would be 1. modify MASShortcut to provide this capability or 2. migrate to a shortcut recorder that has this capability (like Sindre Sorhus KeyboardShortcut library). Then Rectangle would have to be modified to perform the action every interval.

I don't have the bandwidth to do this at the moment, but would welcome a pull request that works on it.
